pkg/sql/codegen/tensorflow/codegen.go

```diff
@@ -36,7 +36,7 @@ Number of epochs the training will run.
 range: [1, Infinity]`, attribute.IntLowerBoundChecker(1, true)},
 	"train.verbose": {attribute.Int, `[default=0]
 Show verbose logs when training.
-possible values: 0, 1`, attribute.IntChoicesChecker([]int{0, 1})},
+possible values: 0, 1`, attribute.IntLowerBoundChecker(0, true)},
```

Comment:
Please take a look at: https://github.com/sql-machine-learning/sqlflow/pull/1385? we can have more levels to show more detailed logs.